scripts/build/Jenkins/Jenkinsfile

```diff
@@ -813,7 +813,7 @@ try {
                         }
                     }
 
-                    pipelineProperties.add(parameters(pipelineParameters))
+                    pipelineProperties.add(parameters(pipelineParameters.unique()))
```

Comment:
is the parameter comparison going to deduplicate if the values are different? if values are different, which one would win?

Reply:
Yeah, it will have duplicated parameter names if the default values are different, and the last value will win. But if we want to give different default values to a parameter, we should use different parameter names.